### ISSUE TITLE
Unknown layer type: Warping 

### ISSUE THREAD

**Issue by kspeng** [Ravi-Garg/Unsupervised_Depth_Estimation#6]
I got the latest version caffe and set WITH_PYTHON_LAYER := 1 in makefile.config. However, I still get the following error:

===

F0325 16:49:50.756420  6840 layer_factory.hpp:81] Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type: Warping (known types: AbsVal, Accuracy, ArgMax, BNLL, BatchNorm, BatchReindex, Bias, Clip, Concat, ContrastiveLoss, Convolution, Crop, Data, Deconvolution, Dropout, DummyData, ELU, Eltwise, Embed, EuclideanLoss, Exp, Filter, Flatten, HDF5Data, HDF5Output, HingeLoss, Im2col, ImageData, InfogainLoss, InnerProduct, Input, LRN, LSTM, LSTMUnit, Log, MVN, MemoryData, MultinomialLogisticLoss, PReLU, Parameter, Pooling, Power, Python, RNN, ReLU, Reduction, Reshape, SPP, Scale, Sigmoid, SigmoidCrossEntropyLoss, Silence, Slice, Softmax, SoftmaxWithLoss, Split, Swish, TanH, Threshold, Tile, WindowData)

===
Is there anything I missed?

**Comment by iamravigarg** [comment]
You have to change the proto file and add the warping layer before recompiling the caffe.

**Comment by kspeng** [comment]
Appreciate it. I will try it.
